modules/reactor/doc/content/source/meshgenerators/AssemblyMeshGenerator.md

```diff
@@ -32,6 +32,32 @@ The `AssemblyMeshGenerator` objects automatically assigns boundary information d
 
 If the assembly is extruded to three dimensions the top-most boundary ID must be assigned using [!param](/Mesh/ReactorMeshParams/top_boundary_id) and will have the name "top", while the bottom-most boundary must be assigned using [!param](/Mesh/ReactorMeshParams/bottom_boundary_id) and will have the name "bottom".
 
+## Metadata Information
+
+Users may be interested in defining additional metadata to represent the reactor geometry and region IDs assigned to each geometry zone, which may be useful to users who want mesh geometry and composition information without having to inspect the generated mesh itself. [!param](/Mesh/AssemblyMeshGenerator/show_rgmb_metadata) can be set to true in order to see the values of these metadata entries as console output. The following metadata is defined on the assembly mesh:
+
+- `assembly_type`: Value of type_id associated with assembly, equivalent to [!param](/Mesh/AssemblyMeshGenerator/assembly_type)
+- `pitch`: Assembly pitch, equivalent to [!param](/Mesh/ReactorMeshParams/assembly_pitch)
+- `is_homogenized`: Whether or not assembly mesh is homogenized, equivalent to [!param](/Mesh/PinMeshGenerator/homogenized)
+- `is_single_pin`: Whether or not assembly mesh is represented by a single pin region or a lattice of pins, equivalent to [!param](/Mesh/PinMeshGenerator/use_as_assembly).
+- `duct_halfpitches`: Location of apothems defining the duct locations, equivalent to [`AssemblyMeshGenerator`](AssemblyMeshGenerator.md)/[!param](/Mesh/AssemblyMeshGenerator/duct_halfpitch)
+- `background_region_id`: 1-D vector of region_ids corresponding to axial zones of background regions of assembly mesh.
+- `duct_region_ids`: 2-D vector of region ids corresponding to radial and axial zones within duct regions of assembly mesh. Inner indexing is radial zones, while outer index is axial zones.
+
+If the assembly is represented as a single pin, the following metadata is also defined:
+
+- `ring_radii`: Location of rings comprising of assembly region, equivalent to [`PinMeshGenerator`](PinMeshGenerator.md)/[!param](/Mesh/PinMeshGenerator/ring_radii).
+- `ring_region_ids`: 2-D vector of region ids corresponding to radial and axial zones within ring regions of assembly mesh. Inner indexing is radial zones, while outer index is axial zones.
```

Comment:
does it correspond to a parameter

Reply:
PinMeshGenerator defines a global vector `region_ids` that includes rings, backgrounds, and ducts, whereas the metadata entry is a subset of this parameter that corresponds to the rings. I've tried to make this distinction in the updated documentation